docs/source/reference/modding/font.rst

```diff
@@ -9,116 +9,35 @@ To our knowledge, the format is undocumented.
 
 OpenMW can load this format and convert it on the fly into something usable 
 (see font loader `source code <https://github.com/OpenMW/openmw/blob/master/components/fontloader/fontloader.cpp#L210>`_). 
-In OpenMW 0.32, an --export-fonts command line option was added to write the converted font 
+You can use --export-fonts command line option to write the converted font
 (a PNG image and an XML file describing the position of each glyph in the image) to the current directory.
 
 TrueType fonts
 --------------
 
 Unlike vanilla Morrowind, OpenMW directly supports TrueType (``.ttf``) fonts.
 
-0.45.0+ way
------------
 This is the recommended way to install replacement fonts.
 
--	To replace the primary "Magic Cards" font:
+	1.	Download `TrueType fonts for OpenMW <https://www.nexusmods.com/morrowind/mods/46854>`_
+	2.	Place the ``Fonts`` folder from archive to the configuration folder. Use :doc:`paths` article to find the folder.
 
-	1.	Download `Pelagiad <https://isaskar.github.io/Pelagiad/>`_ by Isak Larborn (aka Isaskar).
-	2.	Create ``Fonts`` folder at the location of your ``openmw.cfg``.
-	3.	Copy ``openmw_font.xml`` and ``Pelagiad.ttf`` files into the folder.
-	4.	Either remove the ``MonoFont`` entry from the created ``openmw_font.xml`` or copy the ``DejaVuLGCSansMono.ttf`` to the ``Fonts`` folder (prefer latter option on Windows).
-	5.	If desired, you can now delete the original ``Magic_Cards.*`` files from your ``Data Files/Fonts`` directory.
-	6.	Pelagiad glyphs may appear to be pretty small, so open ``settings.cfg`` and adjust the following settings as you deem necessary::
+Now Fonts folder should include ```openmw_font.xml``` file and three ``.ttf``` files.
```

Comment:
it seems to me, here is superfluous `

Reply:
Doesn't really seem to be superfluous to me. It's here just to make sure the user didn't do anything stupid.